doc/source/ray-overview/use-cases.rst

```diff
@@ -57,12 +57,23 @@ Batch inference refers to generating model predictions over a set of input obser
         :text: [Example] Batch OCR processing using Ray Data
         :classes: btn-link btn-block stretched-link
 
+.. _ref-use-cases-mmt:
+
 Many Model Training
 -------------------
 
 Many model training is common in ML use cases such as time series forecasting, which require fitting of models on multiple data batches corresponding to locations, products, etc.
 Here, the focus is on training many models on subsets of a dataset. This is in contrast to training a single model on the entire dataset.
 
+How do I do many model training on Ray?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are three ways of using Ray to express this workload.
+
+1. If you have a large amount of data, use Ray Data (:ref:`Tutorial <mmt-datasets>`).
+2. If you want to integrate with tools, such as wandb and mlflow, and if you have less than 20,000 models, use Ray Tune (:ref:`Tutorial <mmt-tune>`).
+3. If you want lower level control, better scale (up to 1 million models), maybe faster performance, use Ray Core (:ref:`Tutorial <mmt-core>`). Note that this requires you to be more careful about implementation.
```

Comment:
I am a bit confused by this categorization. Are they mutually exclusive? I thought we would want to use Ray Data and Tune together Maybe this is what you mean, but I think it could be more clear; one suggestion is to list Ray Core as a separate paragraph, more like an afterthought instead of equivalent to the other two.

Also confused by the "less than 20,000 models" part. What happens if you have more?

Reply:
RE: Tune vs Ray Data -- we are talking about two different APIs that are mutually exclusive, the data `map_groups` and the Tune standard grid_search sweep. 

RE: 20k - Basically Tune performance starts to degrade close to beyond that.